critiquebrainz/frontend/templates/review/browse.html

```diff
@@ -5,6 +5,16 @@
 
 {% block content %}
 <h2>{{ _('Reviews') }}</h2>
+<form method="GET" action="{{ url_for('review.browse') }}">
+  <div class="form-group" style="display: inline-block;">
+    <select class="form-control input-sm" name="entity_type" onchange="this.form.submit()"> 
```

Comment:
I think it would look better if we used "[nav-pills](https://getbootstrap.com/components/#nav-pills)" here (similar to what we have at https://metabrainz.org/reports). A row that looks like `Entity type: <All> <Release group> <Event> <Place>`.